sssd_test_framework/utils/sbus.py

```diff
@@ -0,0 +1,476 @@
+"""Manage D-Bus operations"""
+
+from __future__ import annotations
+
+import xml.etree.ElementTree as xml
+from abc import ABC
+from typing import Any, Final
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.ssh import SSHProcessResult
+
+from sssd_test_framework.utils.dbus_types import (
```

Comment:
Can you move dbus_types to a subfolder so it is clear that it is not utility itself? And also, please use relative import for our modules. I.e. `from .dbus.dbus_types import ...`

Reply:
Done